src/JoyOI.Monitor/JoyOI.Monitor.csproj

```diff
@@ -4,14 +4,19 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Include="config.json.example" />
+  </ItemGroup>
+
   <ItemGroup>
     <Folder Include="Views\Account\" />
-    <Folder Include="wwwroot\" />
+    <Folder Include="wwwroot\scripts\external\" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.3" />
     <PackageReference Include="JoyOI.UserCenter.SDK" Version="1.0.0-alpha-10073" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.0.0" />
```

Comment:
Please don't use this library, this lib is not a cross-plat lib.

Reply:
removed. don't know how this thing comes in.